data/data/aws/main.tf

```diff
@@ -16,29 +16,24 @@ module "bootstrap" {
   source = "./bootstrap"
 
   ami                      = "${var.aws_ec2_ami_override}"
-  cluster_name             = "${var.cluster_name}"
+  cluster_id               = "${var.cluster_id}"
   ignition                 = "${var.ignition_bootstrap}"
   subnet_id                = "${module.vpc.public_subnet_ids[0]}"
   target_group_arns        = "${module.vpc.aws_lb_target_group_arns}"
   target_group_arns_length = "${module.vpc.aws_lb_target_group_arns_length}"
   vpc_id                   = "${module.vpc.vpc_id}"
   vpc_security_group_ids   = "${list(module.vpc.master_sg_id)}"
 
-  tags = "${merge(map(
-      "Name", "${var.cluster_name}-bootstrap",
-    ), local.tags)}"
+  tags = "${local.tags}"
```

Comment:
See 8a37f72 and f56c341 about not putting `kubernetes.io/cluster/...` on bootstrap resources except the instance.  Do you feel like the accidental-adoption concern is ungrounded?

Reply:
everybidy needs to have ``kubernetes.io/cluster/...`` tag so that destroy captures bootstrap resources.

Reply:
Are we still planning to use the `kubernetes.io/cluster/...` tag for destroy? Since everything is tagged with the `Name` tag, can't we search on that instead?

Reply:
> Are we still planning to use the `kubernetes.io/cluster/...` tag for destroy? Since everything is tagged with the `Name` tag, can't we search on that instead?

`Name` is resource name and has nothing to do with the cluster. AWS will use `kubernetes.io/cluster/...` tag.

Reply:
I'm in favor of keeping a tag that Kubernetes ignores (like `openshiftClusterID`) for tearing down bootstrap resources that are not part of the Kubernetes cluster.  But if folks think we don't have to worry about the cluster accidentally adopting bootstrap resources, I'm ok collapsing to just the `kubernetes.io/cluster/...` tag.